registry/application/services/file_service/constants.py

```diff
@@ -6,31 +6,31 @@
 
 class FileType(Enum):
     ORG_ASSETS = {
-        "file_type": "ASSETS",
+        "file_type": "ORG_ASSETS",
```

Comment:
do these have any impact on UI ?

Reply:
nope. still this is not integrated with UI